### DISCUSSION TITLE
Server response issues

### DISCUSSION THREAD

**Discussion by Dj0ko** [realworld-apps/realworld#704] [Q&A] [answered]
Hello! I have a problem with server response. Recently there was a spam from someone and because of that server started working with some defects.

**Comment by geromegrignon** [comment]
Hello @Dj0ko,

We have a related open issue about that: https://github.com/gothinkster/realworld/issues/700

To sum up, there is no plan to provide a scalable infrastructure for the demo website.
We are considering alternatives to provide everyone a way to run it.

**Comment by geromegrignon** [comment]
Hey @Dj0ko,

We deployed a temporarily new API: https://realworld-temp-api.herokuapp.com/api
Feel free to use it.

**Comment by geromegrignon** [comment, marked as answer]
Hey @Dj0ko,

The existing domain conduit.productionready.io now redirects to a new API deployed on Heroku.
You shouldn't face the server responses latency anymore.
